pcdsdevices/epics_motor.py

```diff
@@ -56,6 +56,8 @@ class PCDSMotorBase(FltMvInterface, EpicsMotor):
     direction_of_travel = Cpt(Signal)
     # This attribute will show if the motor is disabled or not
     disabled = Cpt(EpicsSignal, ".DISP")
+    # This attribute changes if the motor is stopped and unable to move 'Stop', paused and ready to resume on Go 'Paused', and able to move 'Go' or resume a move.
```

Comment:
`travis` is upset because this line is too long (79 characters is the max). This isn't a functional thing, it's just a python coding style thing.